<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureIADPage01</class>
 <widget class="QDialog" name="ConfigureIADPage01">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Assistant</string>
  </property>
  <widget class="QLabel" name="label_caption1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>571</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Assign the activated spectrometer(s) to the measured variables.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_step1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Step 1:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_diffuseTransmission">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Diffuse transmission</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_unscatteredTransmission">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>150</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Select spectrometer</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_diffuseReflection">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Diffuse reflection</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_diffuseTransmission">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>150</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Select spectrometer</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_unscatteredTransmission">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Unscattered transmission</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_diffuseReflection">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>150</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Select spectrometer</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="button_next">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_cancel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>440</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_opticalParameters">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Calculation possible:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_absorptionCoefficient">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>µa</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_scatteringCoefficient">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>µs</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_reducedScatteringCoefficient">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>µs'</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_anisotropy">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>g</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_step2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Step 2:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_caption2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>150</y>
     <width>571</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>IAD arguments</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_incidentLightAngle">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>185</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>90</number>
   </property>
   <property name="value">
    <number>8</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_incidentLightAngle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>185</y>
     <width>105</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Incident light angle:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_incidentLightAngleUnit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>185</y>
     <width>26</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>deg</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_errorTolerance">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>105</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Error tolerance:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_errorTolerance">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.001000000000000</double>
   </property>
   <property name="maximum">
    <double>10.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.001000000000000</double>
   </property>
   <property name="value">
    <double>0.001000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_errorToleranceUnit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>26</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_quadraturePoints">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>235</y>
     <width>105</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string># Quadrature points:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_quadraturePoints">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>235</y>
     <width>60</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>64</number>
   </property>
   <property name="value">
    <number>8</number>
   </property>
   <property name="displayIntegerBase">
    <number>10</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
